### PR TITLE
updated config

### PR DESCRIPTION
**Pull request by hmcts-kodatala**
**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x ] No
```

### CODE REVIEW

**Review by thomast1906 — CHANGES_REQUESTED** [review]
Looks to be values from demo - these are not needed for other envs

File: apps/fis/fis-hmc-api/ithc.yaml [hunk 1]
```diff
@@ -6,7 +6,9 @@ metadata:
 spec:
   values:
     java:
-      replicas: 2
+      ingressClass: traefik-private
+      disableIngressClassAnnotation: true
```
Comment (thomast1906) [inline apps/fis/fis-hmc-api/ithc.yaml:10]: Looks to be values from demo - these are not needed for other envs

File: apps/fis/fis-hmc-api/perftest.yaml [hunk 2]
```diff
@@ -7,7 +7,9 @@ spec:
   releaseName: fis-hmc-api
   values:
     java:
-      replicas: 2
+      ingressClass: traefik-private
+      disableIngressClassAnnotation: true
+      image: hmctspublic.azurecr.io/fis/hmc-api:prod-2a50409-20230126163420 #{"$imagepolicy": "flux-system:demo-fis-hmc-api"}
```
Comment (hannah38) [inline apps/fis/fis-hmc-api/perftest.yaml:12]:
```suggestion
      image: hmctspublic.azurecr.io/fis/hmc-api:prod-2a50409-20230126163420 #{"$imagepolicy": "flux-system:perftest-fis-hmc-api"}
```

File: apps/fis/fis-hmc-api/ithc.yaml [hunk 3]
```diff
@@ -6,7 +6,9 @@ metadata:
...
+      image: hmctspublic.azurecr.io/fis/hmc-api:prod-2a50409-20230126163420 #{"$imagepolicy": "flux-system:demo-fis-hmc-api"}
```
Comment (hannah38) [inline apps/fis/fis-hmc-api/ithc.yaml:11]:
```suggestion
      image: hmctspublic.azurecr.io/fis/hmc-api:prod-2a50409-20230126163420 #{"$imagepolicy": "flux-system:ithc-fis-hmc-api"}
```

File: apps/fis/fis-hmc-api/ithc.yaml [hunk 4]
```diff
@@ -6,7 +6,7 @@ metadata:
 spec:
   values:
     java:
-      replicas: 2
+      image: hmctspublic.azurecr.io/fis/hmc-api:prod-2a50409-20230126163420 #{"$imagepolicy": "flux-system:perftest-fis-hmc-api"}
```
Comment (hannah38) [inline apps/fis/fis-hmc-api/ithc.yaml:9]:
```suggestion
      image: hmctspublic.azurecr.io/fis/hmc-api:prod-2a50409-20230126163420 #{"$imagepolicy": "flux-system:ithc-fis-hmc-api"}
```

File: apps/fis/automation/kustomization.yaml [hunk 5]
```diff
@@ -11,4 +11,6 @@ resources:
   - ../fis-ds-update-web/image-policy.yaml
   - ../fis-hmc-api/image-repo.yaml
   - ../fis-hmc-api/image-policy.yaml
-  - ../fis-hmc-api/demo-image-policy.yaml
\ No newline at end of file
+  - ../fis-hmc-api/demo-image-policy.yaml
+  - ../fis-hmc-api/perftest-image-policy.yaml
+   - ../fis-hmc-api/ithc-image-policy.yaml
```
Comment (hannah38) [inline apps/fis/automation/kustomization.yaml:16]: Indentation is off here

File: apps/fis/fis-hmc-api/demo.yaml [hunk 6]
```diff
@@ -7,3 +7,6 @@ spec:
     java:
       ingressClass: traefik-private
       disableIngressClassAnnotation: true
+      image: hmctspublic.azurecr.io/fis/hmc-api:prod-2a50409-20230126163420 #{"$imagepolicy": "flux-system:demo-fis-hmc-api"}
+      environment:
+          DUMMY_VAR_TO_REDEPLOY: 1
```
Comment (hannah38) [inline apps/fis/fis-hmc-api/demo.yaml:12]: Failing checks here because of indentation "expected 8 but found 10"

**Review by hannah38 — APPROVED** [review]

**Review by hmcts-shajahan — APPROVED** [review]

**Pull request merged**